src/FilterConditionSelector/FilterConditionSelector.tsx

```diff
@@ -60,15 +61,17 @@ const FilterConditionSelector: React.FC<Props & WithOutsideClickProps> = ({
 	}, []);
 
 	const handleDisableBodyScroll = (e: React.WheelEvent) => {
-		e.preventDefault();
+		if (checkBrowser("Safari") && !checkBrowser("Chrome")) {
```

Comment:
Может обойтись одним `checkBrowser("Safari")`?
```suggestion
		if (checkBrowser("Safari")) {
```

Reply:
я так сделал потому что хром выдает вот так
`"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36"`